plugins/include/amxmodx.inc

```diff
@@ -2321,9 +2322,35 @@ native md5(const szString[], md5buffer[34]);
  * @return              Number of cells written to the buffer (always 32)
  * @error               If the file can not be opened, and error is thrown.
  */
+#pragma deprecated Use now hash_file() function. See INC files for more info.
 native md5_file(const file[], md5buffer[34]);
 
 /**
```

Comment:
Keep it consistent. 

* Use spaces for alignment.
* About `type` description, I would say: `Hash type, see Hash_* constants in amxconst`